source/_integrations/dsmr_reader.markdown

```diff
@@ -21,21 +23,18 @@ The `dsmr_reader` sensor integration allows you to easily add all sensors that [
 - DSMR Reader
 - MQTT broker
 
-## Setup
+## Preparation
 
-1. Configure the MQTT broker in DSMR Reader which Home Assistant also connects to
-2. Enable the following data sources in DSMR Reader with the default mapping:
+1. Add the MQTT broker integration in Home Assistant, if you haven't done so already
+2. Configure the MQTT broker in the DSMR Reader application
+3. Enable the following data sources in the DSMR Reader administration pages with the default mappings:
    - Day consumption: Split topic
    - Gas consumption: Split topic
    - Meter Statistics: Split topic
    - Telegram: Split topic
 
-## Configuration
+{% include integrations/config_flow.md discovery="true" %}
```

Comment:
That value should be extracted from the meta data.

```suggestion
{% include integrations/config_flow.md %}
```

Reply:
I see that the config_flow snippet supports `ha_dhcp` , `ha_homekit`, `ha_ssdp` and `ha_zeroconf` to show the text about automatic discovery. I presume it is fine that I add `ha_mqtt` to that snippet? Because otherwise it doesn't detect the automatic discovery aspect.

Reply:
Sounds good 👍